conda/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$PREFIX" ]; then
```

Comment:
When would this ever occur?

Reply:
If you called the `conda/build.sh` script by hand. It's just so the script is modestly more robust.